template.md

```diff
@@ -0,0 +1,51 @@
+<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="" />
+
+## Agenda for the TBD meeting of Ecma TC39
+
+- **Host**: TBD
+- **Dates**: TBD
+- **Times**:
+  - 10:00 to 17:00 PDT on TBD
+  - 10:00 to 16:00 PDT on TBD
+- **Location**: TBD
+- **Wifi**: TBD
+- **Dinner**: TBD
+- **Contact**:
+  - Name: TBD
+  - Phone: TBD
+  - Email: TBD
+
+## Logistics
+
+TBD Host Logistics (Parking, building access, etc.)
+
+### Registration
+
+TBD
+
+### Hotels
+
+TBD Hotels nearby (optional)
+
+## Agenda items
+
+1. Opening, welcome and roll call
+  1. Opening of the meeting (Mr. Neumann)
+  1. Introduction of attendees
+  1. Host facilities, local logistics
+1. Adoption of the agenda
+1. Approval of the minutes from last meeting
+1. Report from the Ecma Secretariat (15m)
+1. ECMA262 Status Updates (15m)
+1. ECMA402 Status Updates (15m)
+1. Test262 Status Updates (15m)
+1. Timeboxed agenda items: topics are timeboxed in increments of 15 minutes up to 60 minutes. Shorter proposals come first. Additional discussion comes at the end of the meeting.
```

Comment:
I think it's sufficient to simply order by timebox size since the timeboxed items will be gone through quickly. Since we're pretty much assured to make it through all the timeboxed items, ordering by topic type doesn't seem very useful to me. I don't mind much though, so I'll change it if no one agrees with me!

Regarding the timebox limit, my feeling is that if you can't say that your discussion will be <= 1 hour then you really don't have much idea how long the discussion should take and so you should just wait for your turn for a non-timeboxed topic. In other words, the uncertainty of how long a discussion will need to take is very high once you get beyond an hour. But again, I don't mind much :)